### PR TITLE
[usecase][linux] Add tests checking alert/confirm/prompt dialogs

### PR DESCRIPTION
**Pull request by yhe39**
BUG=https://crosswalk-project.org/jira/browse/XWALK-5828

### CODE REVIEW

**Comment by yhe39** [comment]
@zqzhang please review my pr, thanks

File: usecase/usecase-wrt-linux-tests/samples/AlertDialog/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Usecase Design
+
+This sample demonstrates Web Notification feature basic functionalities, include:
+
+* Notification constructor, requestPermission/permission and onclose
+
+## Spec Link
+
+https://crosswalk-project.org/jira/browse/XWALK-4061
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/AlertDialog/README.md]: ?? Seems you are not doing for this, please update it.

Also please use "BUG=" in the commit message. See https://crosswalk-project.org/contribute/contributing-code.html

File: usecase/usecase-wrt-linux-tests/samples/AlertDialog/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/AlertDialog/index.html]: s/2015/2016/g

File: usecase/usecase-wrt-linux-tests/samples/AlertDialog/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Liu.Xin <xinx.liu@intel.com>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/AlertDialog/index.html]: Also use your name here.

File: usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/index.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Runtime Test: ALert Dialog</title>
+    <!--script src="js/main.js"></script-->
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/index.html]: Please remove commented code.

**Comment by yhe39** [comment]
@zqzhang please help to review my modified files, thanks!

File: usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/index.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <meta charset="utf-8">
+    <title>Runtime Test: ALert Dialog</title>
+  </head>
+  <body>
+    <button id="alert">alert("Hello")</button>
+    <button id="confirm">confirm("Hello")</button>
+    <button id="prompt">prompt("Hello")</button>
+
+    <script>
+        document.querySelector('#alert').addEventListener('click', event => {
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/index.html]: Indent not consistent in this script.

File: usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/manifest.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "runtime_webnotification",
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/manifest.json]: s/runtime_webnotification/runtime_alertdialog/

File: usecase/usecase-wrt-linux-tests/samples/AlertDialog/README.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+## Usecase Design
+
+This sample demonstrates popup alert/confirm/prompt dialog functionalities, include:
+
+
+## Spec Link
+
+BUG=XWALK-5819
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/AlertDialog/README.md]: https://crosswalk-project.org/jira/browse/XWALK-5819

File: usecase/usecase-wrt-linux-tests/samples/AlertDialog/index.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+<meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, width=device-width">
+<link rel="stylesheet" href="../../css/bootstrap.css">
+<link rel="stylesheet" href="../../css/main.css">
+<script src="../../js/jquery-2.1.3.min.js"></script>
+<script src="../../js/bootstrap.min.js"></script>
+<script src="../../js/common.js"></script>
+<script src="../../js/tests.js"></script>
+
+<div id="header">
+  <h3 id="main_page_title"></h3>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/samples/AlertDialog/index.html]: Any text for the head?

File: usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/index.html [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+      alert('Hello');
+      });
+      document.querySelector('#confirm').addEventListener('click', event => {
+      confirm('Hello');
+      });
+      document.querySelector('#prompt').addEventListener('click', event => {
+      prompt('Hello');
+      });
+    </script>
+    <style>
```
Comment (zqzhang) [inline usecase/usecase-wrt-linux-tests/res/testapp/alertdialog/index.html]: Put this in `<head>`

**Comment by zqzhang** [comment]
Thanks, LGTM.

**Pull request merged**